src/docs/integrations/jira/index.mdx

```diff
@@ -1,18 +1,27 @@
 ---
-title: "Jira Integration"
+title: 'Jira Integration'
 ---
 
-Create an [Atlassian Cloud Instance](http://go.atlassian.com/cloud-dev)
+Create an [Atlassian Cloud Developer Instance](http://go.atlassian.com/cloud-dev)
 
-Configure Jira settings.
-  1. Navigate to Jira settings (cog icon) > Apps > Manage apps.
-  2. Scroll to the bottom of the Manage apps page, and click Settings (1).
-  3. Select Enable development mode and Enable private listings (2), and click Apply.
+Atlassian offers a development-specific org option, different from simply having a personal Jira organization. Even if you already have a personal Jira org, it's worth creating a development org, because it will correctly set a number of necessary options for you automatically.
+
+Set Up ngrok
+
+(This is only necessary if you're developing locally.) [Download](https://ngrok.com/download) and [set up](https://ngrok.com/docs/getting-started/) ngrok, then start an ngrok server.
+
+Configure Jira Settings
+
+1. Create a new project in Jira.
```

Comment:
This is only necessary if you are using a brand new Jira organization, right? If you've already been using Jira and have a project/projects you can skip this step.

Reply:
True. But we're also telling people to make a new cloud instance when they might already have one. That said, I'm fine to change this to, "If you don't have any projects in your Jira org yet, create a new project."